xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-core/src/main/resources/conf/managed-schema

```diff
@@ -163,6 +165,10 @@
     <!-- Whether the entity is hidden on not. Only documents can be made hidden explicitly.
          The attachments, objects and object properties are hidden if the document that holds them is hidden. -->
     <field name="hidden" type="boolean" indexed="true" stored="true" />
+    <!-- The references of the resources where the various links found in that entity are leading to -->
+    <field name="links" type="string" indexed="true" stored="true" multiValued="true" />
+    <!-- Contains links plus all the references parents to make easier to search for links from any entity level (per wiki, per space, per document, etc.) -->
+    <field name="links_extended" type="string" indexed="true" stored="false" multiValued="true" />
```

Comment:
I don't recall, what does `stored="false"` mean?

Reply:
That it's not stored :)

Basically it means you indicate Solr that you will only use it as a search criteria but you never want to retrieve it.

Reply:
Ok so it only stores indexes.

Reply:
Yes, not exactly sure what it means exactly internally, but from user point it feels like it.